multiprocessing.TimeoutError is missing in Python 3 stubs

```python
import time
import multiprocessing


def func(x):
    # type: (int) -> int
    time.sleep(x)
    return x


def main():
    # type: () -> None
    pool = multiprocessing.Pool(2)
    ite = pool.imap(func, range(10), chunksize=1)

    try:
        while True:
            x = ite.next(timeout=3)
            print(x)
    except multiprocessing.TimeoutError:
        print('timeout')
    

if __name__ == '__main__':
    main()
```

This code correctly works as follows.

```
$ python --version
Python 3.6.4
$ python foo.py
0
1
2
3
4
5
6
timeout
```

However, `mypy` (GitHub master) fails as follows:

```
$ mypy --version
mypy 0.570-dev-c594761aa1dbe76a3a0bc199b03fbf684ce588d8
$ mypy --python-version 3.6 foo.py
foo.py:18: error: "Iterable[Any]" has no attribute "next"
foo.py:20: error: Module has no attribute "TimeoutError"
```

FYI: when we specify `--python-version 2.7`, `mypy` says nothing against the code.

--- a/stdlib/3/multiprocessing/__init__.pyi
+++ b/stdlib/3/multiprocessing/__init__.pyi
@@ -9,7 +9,7 @@ from logging import Logger
 from multiprocessing import connection, pool, synchronize
 from multiprocessing.context import (
     BaseContext,
-    ProcessError, BufferTooShort, TimeoutError, AuthenticationError)
+    ProcessError as ProcessError, BufferTooShort as BufferTooShort, TimeoutError as TimeoutError, AuthenticationError as AuthenticationError)
 from multiprocessing.managers import SyncManager
 from multiprocessing.process import current_process as current_process
 from multiprocessing.queues import Queue as Queue, SimpleQueue as SimpleQueue, JoinableQueue as JoinableQueue

